### PR TITLE
fix(optimum): rename prefix_block_size to sub_block_size to sync with vllm-path

### PR DESCRIPTION
**Pull request by rebel-eunji**
### 🚀 Summary of Changes

optimum-path previously used prefill_block_size to determine the granularity of prefix cache hits.
However, vllm-path recently introduced sub_block_size for the same purpose, so optimum-path now follows that convention for consistency.

---

### 📌 Related Issues / Tickets

* Resolves #
* Related to #

---

### ✅ Type of Change

* [ ] 🚀 Release (`release`)
* [ ] ✨ Feature (`feature`)
* [ ] 🧠 Model support (`model`)
* [ ] 🧬 Core engine changes (`core`)
* [ ] 🛠 Bug fix (`fix`)
* [ ] ⚙️ Performance improvement (`perf`)
* [ ] 🔁 Refactor or code cleanup (`refactor`)
* [ ] 📄 Documentation (`docs`)
* [ ] ❓ Other (`other`): please describe

---

### 🧪 How to Test

1. Run `...`
2. Verify output: `...`
3. Edge case tested: `...`

---

### 📸 Screenshots / Logs (if applicable)

---

### 📋 Checklist

* [ ] PR title follows Conventional Commits format
* [ ] This PR is linked to an existing issue
* [ ] The test method is described, and the expected result is clearly stated
* [ ] Relevant documentation has been updated (if applicable)

---

### 💬 Notes

---

### CODE REVIEW

File: vllm_rbln/config.py [hunk 1]
```diff
@@ -188,8 +188,8 @@ class OptimumRBLNConfig(RBLNConfigBase):
     derives from the vLLM settings when the model is compiled. With a
     pre-compiled model only the `device` entries apply."""
 
-    prefix_block_size: int | None = None
-    """Block size of the prefix cache. Defaults to the prefill chunk size."""
+    sub_block_size: int | None = None
```
Comment (rebel-jonghewk) [inline vllm_rbln/config.py]: If these mean the same thing, is there a reason not to move this to `RBLNConfigBase` ?

Comment (rebel-eunji) [reply vllm_rbln/config.py]: I thought I should keep it for the old name. But I realized I don't have to that.
Thanks!

Comment (rebel-eunji) [reply vllm_rbln/config.py]: https://github.com/RBLN-SW/vllm-rbln/pull/1164/changes/31050ab4798610283287979a47f7a91f8de93335
